palantirnet/the-build
# DEV-71: Replace drupal-check with phpstan

Replaces drupal-check with phpstan.

See https://palantir.atlassian.net/browse/DEV-71

### Testing instructions

```
composer require --dev palantirnet/the-build:dev-DEV-71-replace-drupal-check-with-phpstan
vendor/bin/the-build-installer
```

* This should result in adding a `phpstan.neon` file in your project root. In the new file, there should be include paths with the correct web root for your project.
* Running `vendor/bin/phing code-review` should now include phpstan and not include drupal-check
* Running `vendor/bin/phpstan analyse` should do the same thing (use the new `phpstan.neon` to scan custom code)
